django/test/testcases.py

```diff
@@ -524,17 +524,23 @@ def assertContains(
             response, text, status_code, msg_prefix, html
         )
 
+        if response.streaming:
+            content = b"".join(response.streaming_content)
+        else:
+            content = response.content
```

Comment:
```suggestion
        content = b"".join(response.streaming_content) if response.streaming else response.content
```

This is merely a suggestion but I've seen reviewers recommend "condition expression" 😊  Not saying change it - just an option!

Reply:
Sorry to bikeshed, can't help myself 🤣

A possibly more pythonic option could be:

```suggestion
        try:
            content = response.content
        except AttributeError:
            content = b"".join(response.streaming_content)
```